pkg/occlient/occlient.go

```diff
@@ -753,6 +754,9 @@ func (c *Client) GetImageStream(imageNS string, imageName string, imageTag strin
 		}
 		if e != nil && err != nil {
 			// Imagestream not found in openshift and current namespaces
+			if experimental.IsExperimentalModeEnabled() {
+				return nil, fmt.Errorf("component \"%s\" not found", imageName)
```

Comment:
You could use `%q` to avoid escaping the `"` character.

Reply:
@dharmit good to know about %q, updated 🙂